bundles/org.openhab.binding.somfymylink/src/main/java/org/openhab/binding/somfymylink/internal/model/SomfyMyLinkCommandSceneSet.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.somfymylink.internal.model;
+
+/**
+ * @author Chris Johnson - Initial contribution
+ */
+public class SomfyMyLinkCommandSceneSet extends SomfyMyLinkCommandSceneBase {
+    public SomfyMyLinkCommandSceneSet(int sceneId, String auth) {
+        super(sceneId, "mylink.scene.run", auth);
+    }
+}
```

Comment:
Some of our build tools don't like EOF without a newline. So add a newline here.